terminologies/handedness/mixedHandedness.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+  "@id": "https://openminds.ebrains.eu/controlledTerms/handedness/mixedHandedness",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Handedness",
+  "name": "mixed handedness",
+  "definition": "Handedness where the organism exhibits dominance in the use of right or left hand or foot differently in the performance of different tasks that require one hand or foot or a dominant hand or foot.",
```

Comment:
I would shorten that to "Handedness where the organism exhibits dominance in the use of right or left hand or foot depending on the task that has to be performed"